test/taglib-lookup/fixtures/declared-and-dynamic-attrs/marko.json

```diff
@@ -2,6 +2,9 @@
     "<test-dynamic-attribute>": {
         "renderer": "./renderer.js",
         "@foo": "string",
-        "@*": "boolean"
+        "@*": {
```

Comment:
I'm surprised that none of these changes broke any tests, are none of them relying on `input["*"]`?

Reply:
Yep, they did not fail. 
This seems like it's not a real good test (and the template had a TBD in it from 3 years ago):
```
    var attr = lookup.getAttribute("test-dynamic-attribute", "DYNAMIC");
    expect(attr != null).to.equal(true);
    expect(attr.type).to.equal("boolean");
    expect(attr.name).to.equal("*");

    attr = lookup.getAttribute("test-dynamic-attribute", "foo");
    expect(attr != null).to.equal(true);
    expect(attr.type).to.equal("string");
```